pkg/apis/node.harvesterhci.io/v1beta1/ksmtuned.go

```diff
@@ -107,4 +107,7 @@ type KsmtunedParameters struct {
 	Decay     uint   `json:"decay"`
 	MinPages  uint   `json:"minPages"`
 	MaxPages  uint   `json:"maxPages"`
+
+	// +kubebuilder:validation:Maximum=1
+	MergeAcrossNodes uint `json:"mergeAcrossNodes"`
```

Comment:
users won't be able to modify it if the parameters are not displayed? should we move this to the upper spec?